packages/flutter/lib/src/widgets/autofill.dart

```diff
@@ -225,3 +228,142 @@ class _AutofillScope extends InheritedWidget {
   @override
   bool updateShouldNotify(_AutofillScope old) => _scope != old._scope;
 }
+
+/// An [InheritedWidget] that configures its descendant [Navigator]s and [Form]s,
```

Comment:
Nit: I think there should be no comma at the end of this line, but it's fine either way.